src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Your App component is function-based whereas it should be class-based, kindly fix refactor it to be class-based.